meta/signature/perturbation/Phenotype.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$id": "/@dcic/signature-commons-schema/meta/signature/perturbation/phenotype.json",
```

Comment:
This filename should be lowercase like the other files, currently it is `Phenotype.json`